src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -2093,6 +2105,13 @@
     </Field>
     <Field Name="Pattern" Type="PatternSyntax" />
   </Node>
+  <Node Name="SlicePatternSyntax" Base="PatternSyntax">
+    <Kind Name="SlicePattern"/>
+    <Field Name="DotDotToken" Type="SyntaxToken">
+      <Kind Name="DotDotToken"/>
+    </Field>
+    <Field Name="Pattern" Type="PatternSyntax" Optional="true"/>
```

Comment:
Is there an example for what this field is being used for? The proposal implied that nothing can be listed after the .. in the list or that there was still an open question there.

Reply:
> The proposal implied that nothing can be listed after the .. in the list 

That's true only for enumerable and multi-dimensional arrays. 

For compatible types this optionally matches the returned slice. See https://github.com/dotnet/csharplang/blob/main/proposals/list-patterns.md#lowering-on-countableindexeablesliceable-type